README.md

```diff
@@ -152,3 +152,17 @@ Built on: TypeScript, JavaScript, HTML etc.
 https://github.com/novuhq/novu
 
 Issues for first timers: https://github.com/novuhq/novu/contribute
+
+
+---
+
+## Watchflix
+
+> Movie recomendation platform that gets data from The Movie Database (TMDB) API.
+
+Built on: JavaScript
+
+https://github.com/carlagesa/Watchflix
+
+Issues for first timers: https://github.com/carlagesa/Watchflix/blob/main/CONTRIBUTING.md
```

Comment:
Thank you. Is there a particular label that you use for first timers?

Reply:
At the moment no, I'm a first timer myself. As we progress we'll figure out  how to be better.